IO/Logging/Logger.cs

```diff
@@ -1,29 +1,61 @@
-﻿namespace LocalAdmin.V2.IO.Logging
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+
+namespace LocalAdmin.V2.IO.Logging
 {
-    /*
     public static class Logger
     {
-        public static void Log(string text)
+        public static StreamWriter? Writer { get; private set; }
+        private static object _lock = new object();
```

Comment:
This should be readonly

Reply:
I copied it from https://github.com/northwood-studios/LocalAdmin-V2/blob/0b0991dab9aa3cb7e01c569052173eb815568a8c/IO/ConsoleUtil.cs#L9 but ok

Reply:
What should be name (especially casing) of it then? :/

Reply:
Ask @zabszk about naming conventions

Reply:
Also, isn't a StreamWriter nullable by default?